### ISSUE TITLE
Diffuse interlobular septal thickening (DIST)

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6569]
**Preferred term label:** Diffuse interlobular septal thickening (DIST)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Diffuse interlobular septal thickening (DIST) is a lung disease pattern found on high-resolution thoracic CT scanning (HRCT or CTPA). It represents pathology in the periphery of the pulmonary lobules (i.e., the interlobular septa). 

**Parent term (use hpo.jax.org/app)**
Interlobular septal thickening on pulmonary HRCT HP:0030879

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Pulmonary veno-occlusive disease (PVOD) OMIM:265450

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
The discussion was to not add this term. Please instead use the term
Interlobular septal thickening on pulmonary HRCT
https://hpo.jax.org/app/browse/term/HP:0030879
together with the modifer 'diffuse' if necessary
https://hpo.jax.org/app/browse/term/HP:0020034
this is because the Fleischer society recommends the current 3 children of https://hpo.jax.org/app/browse/term/HP:0030879.
